.chloggen/tempo-query_ReadinessProbe.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

Comment:
is this enhancement or bug_fix?

Reply:
From my point of view its a pure enhancement. Everything works as before. We only support the kubernetes readiness check.